spec/InformationStructure/schemas/11_Controller.yaml

```diff
@@ -17,18 +17,20 @@ control-construct:
               'controllerName
               candidate: from [DDM://v1/regard-controller$request.body#controller-name]
               running: from candidate
-              operational: from [CDM://v1/provide-list-of-controllers/controller-name], delete entire CC from list if not in response'
+              operational: from running; entire CC to be deleted, if applicationName not included in [[CDM://v1/provide-list-of-controllers/controller-name]$response.body#list-of-controller-names]'
```

Comment:
As per the [ControllerDomainManager+services.yaml](https://github.com/openBackhaul/ControllerDomainManager/blob/develop/spec/ControllerDomainManager%2Bservices.yaml), the CDM service /v1/list-controllers should be called. Correct me if I m wrong. Thanks.